extensions/parquet/table/src/main/java/io/deephaven/parquet/table/ParquetTools.java

```diff
@@ -90,7 +94,7 @@ private ParquetTools() {}
      * @see ParquetFlatPartitionedLayout
      */
     public static Table readTable(@NotNull final String sourceFilePath) {
-        return readTableInternal(new File(sourceFilePath), ParquetInstructions.EMPTY);
+        return readTableInternal(convertToURI(sourceFilePath), ParquetInstructions.EMPTY);
```

Comment:
I think we'll want to find a better name instead of `filePath`, `sourceFilePath` and update documentation accordingly. Ditto for everything else that flows into convertToURI.